ccd-definition/State.json

```diff
@@ -22,5 +22,13 @@
     "Name": "Deleted",
     "Description": "Deleted case state - all data is removed",
     "DisplayOrder": 3
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "Gatekeeping",
+    "Name": "Gatekeeping",
+    "Description": "Gatekeeping case state - when send to gatekeeper event is triggered",
+    "DisplayOrder": 4
```

Comment:
Wouldn't it be better to have Gatekeeping state above deleted? What's the business take on it?